### ISSUE TITLE
Record times for data

### ISSUE THREAD

**Issue by adampbeardsley** [epic-astronomy/LWA_EPIC#24]
@HaraKrish could you summarize what we do and do not know about the options for time recording?

**Comment by HaraKrish** [comment]
The reference epoch is for some reason fixed to 1970-1-1 00:00 UT and when we run EPIC, if we do not pass a start time argument, the default start time is also set to 1970-1-1 00:00 UT. And with the above the time tag is estimated. This is my understanding of what's happening in there with the time tag
